stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json

```diff
@@ -0,0 +1,332 @@
+{
+  "timestamp": [
+    {
+      "key": "first_observed",
+      "transformer": "EpochToTimestamp"
+    },
+    {
+      "key": "last_observed",
+      "transformer": "EpochToTimestamp"
+    }
+  ],
+  "device_ip": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "host_ip"
+    }
+  ],
+  "device_natIp": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "nat_ip"
+    }
+  ],
+  "dns_replyIp": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dns_ip"
+    }
+  ],
+  "dstDevice_ip": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "network-traffic.dst_ref",
+      "object": "nt",
+      "references": "src_ip"
+    },
+    {
+      "key": "x-oca-event.network_ref",
+      "object": "event",
+      "references": "nt"
+    }
+  ],
+  "srcDevice_ip": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "src_ip"
+    },
+    {
+      "key": "x-oca-event.network_ref",
+      "object": "event",
+      "references": "nt"
+    }
+  ],
+  "dstPort": [
+    {
+      "key": "network-traffic.dst_port",
+      "object": "nt"
+    }
+  ],
+  "srcPort": [
+    {
+      "key": "network-traffic.src_port",
+      "object": "nt"
+    }
+  ],
+  "ipProtocol": [
+    {
+      "key": "network-traffic.protocols",
+      "object": "nt",
+      "transformer": "FormatTCPProtocol"	
+    }
+  ],
+  "dstDevice_natIp": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_nat"
+    }
+  ],
+  "srcDevice_natIp": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_nat"
+    }
+  ],
+  "device_mac": [
+    {
+      "key": "mac-addr.value",
+      "object": "host_mac"
+    }
+  ],
+  "srcDevice_mac": [
+    {
+      "key": "mac-addr.value",
+      "object": "src_mac"
+    },
+    {
+      "key": "ipv4-addr.resolves_to_ref",
```

Comment:
Should be plural; i.e. `resolves_to_refs`

Reply:
Fixed it too, even the one below